src/main/app/copy-icons.js

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env node
+const fs = require('fs-extra');
+
+// Async with promises: 
+fs.copy('node_modules/@mdi/svg/svg', '../resources/static/icons/svg')
+    .then(() => console.log('Icons copied successfully!'))
+    .catch(err => console.error(err));
```

Comment:
This is what you mean with "cp"?

I checked the build with Windows 10, it works fine. I will now check with MacOS.

Reply:
"cp" is not used anymore. This is copied production code.

Reply:
Fine, I did not find "cp", I am checking with MacOS anyways :)